src/MCPServer/lib/linkTaskManagerFiles.py

```diff
@@ -45,40 +45,39 @@ def __init__(self, jobChainLink, pk, unit):
         self.tasksLock = threading.Lock()
         self.exitCode = 0
         self.clearToNextLink = False
-        sql = """SELECT * FROM StandardTasksConfigs where pk = '%s'""" % (pk.__str__())
-        c, sqlLock = databaseInterface.querySQL(sql)
-        row = c.fetchone()
-        while row != None:
-            filterFileEnd = deUnicode(row[1])
-            filterFileStart = deUnicode(row[2])
-            filterSubDir = deUnicode(row[3])
-            requiresOutputLock = row[4]
-            self.standardOutputFile = deUnicode(row[5])
-            self.standardErrorFile = deUnicode(row[6])
-            self.execute = deUnicode(row[7])
-            self.arguments = deUnicode(row[8])
-            row = c.fetchone()
-        sqlLock.release()
-        if requiresOutputLock:
+
+        stc = StandardTaskConfig.objects.get(id=str(pk))
+        # These three may be concatenated/compared with other strings,
+        # so they need to be bytestrings here
+        filterFileEnd = str(stc.filter_file_end) if stc.filter_file_end else ''
+        filterFileStart = str(stc.filter_file_start) if stc.filter_file_start else ''
+        filterSubDir = str(stc.filter_subdir) if stc.filter_subdir else ''
```

Comment:
We only use those properties here and in linkTaskManagerDirectories. I don't think ltmDirectories can get unusual paths like we can in ltmFiles.

(The properties are also assigned, but never used, in ltmGetMicroserviceGeneratedListInStdOut)